docs/_static/searchtester.js

```diff
@@ -0,0 +1,143 @@
+let breadcrumbMapping; // Storing the fetched JSON here
+let resultHeadings = []; // Store the result headings once fetched
+let counter = 0;
+
+// Function to load JSON into script
+async function fetchBreadcrumbMapping() {
+  try {
+    const response = await fetch("/_static/heading_to_breadcrumb_mapping.json");
+    breadcrumbMapping = await response.json();
+  } catch (error) {
+    console.error("Error fetching mapping:", error);
+  }
+}
+
+fetchBreadcrumbMapping();
+
+// Function to create breadcrumbs so that we can just put in title of results
+async function createBreadcrumbs(resultHeading) {
+  try {
+    // Check if the heading exists in the mapping
+    const breadcrumbs =
+      breadcrumbMapping["heading_to_breadcrumb"][resultHeading];
+    if (!breadcrumbs) {
+      throw new Error("No breadcrumbs found for the heading: ", resultHeading);
+    }
+    return breadcrumbs;
+  } catch (error) {
+    console.error("Error fetching mapping:", error);
+    return [];
+  }
+}
+
+// Function to add breadcrumbs above each result heading
+function addBreadcrumbsToResults() {

```

Comment:
Actually, I do not have a better approach than that for now.
Working on the DOM directly like this is far from ideal, it is difficult to maintain and might not be stable over time (typically, there is guarantee that the CSS selector using the `svelte-1yttzcg` class will work for ever, this classname is generated automatically by the Svelte compiler).
I need to think about how I could make the Nuclia Widget more customisable.